server/test/unit/server/db/test_model.py

```diff
@@ -237,6 +237,11 @@ class TestUnit(model.SharedContentUnit):
     def test_abstract(self):
         unit = TestSharedContentUnit.TestUnit()
         try:
+            unit.storage_provider
```

Comment:
would not it be better to split this test into two tescases- test_storage_provider and test_storage_id? to avoid 2 try/except that are testing 2 different things.